js/app/Data/CartoDBInfoWindow.js

```diff
@@ -78,7 +77,9 @@ define([
          obj
         );
 
-        cb($(popupHtml).find('.cartodb-popup-content'));
+        var html = $(popupHtml).find('.cartodb-popup-content');
+
+        cb(html, data);
```

Comment:
You are absolutely right. I'll change it so that the result includes both the html and the data as fields instead, and make it a single-arg callback again.